iam/api/QuickStartV2Test/QuickStartTest.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />

```

Comment:
When you include the junit test logger on runTests.ps1 you'll also need to include a reference to the package here. Take a look at the *.csproj in the KMS samples test folder to see how.

Reply:
Done